src/pages/designing/figma-kit.mdx

```diff
@@ -5,22 +5,162 @@ description: The new Figma design kit contains type styles, color tokens, and co
+
+## Using the Figma kit
+
+Our Figma resources are set up to help you design using key IBM assets. Learn more about using the [2x Grid](#2x-grid), [Components](#components), [Text styles](#text-styles), and [Color styles](#color-styles) in Figma.
+
+### 2x Grid
+
+The IBM 2x Grid is the framework for laying out all visual elements.
+
+#### Understanding the grid
+
+For a detailed walkthrough of the 2x Grid, see the foundational 2x Grid guidance on the Carbon Design System site. The [Overview](https://www.carbondesignsystem.com/guidelines/2x-grid/overview) provides the background and design rationale for the 2x Grid. [Implementation](https://www.carbondesignsystem.com/guidelines/2x-grid/implementation) will get you up and running, designing with the grid.
+
+For information about using the 2x Grid with web pages, including best practices, see our [Layout guidelines](../guidelines/layout).
+
+#### Start using the grid
+
+In Figma, frames function as artboards and containers for everything from screens to buttons. You can learn more about frames in Figma’s <a href="https://help.figma.com/hc/en-us/articles/360041539473-Frames-in-Figma" target="_blank" rel="noopener noreferrer">documentation</a>. You can apply the grid template to a frame in Figma if you have one or more of the Carbon for IBM.com libraries active:
+
+#### To apply the grid template to a frame in Figma:
+
+1. Have one or more of the Carbon for IBM.com libraries active.
+2. Select the frame to apply the template to.
+3. In `Design panel → Layout Grid`, click `Style`.
+4. Choose your breakpoint to display the grid.
+
+You can click the eye icon to hide or show the grid.
+
+![Adding components from a library](../../images/get-started/Designing_Grid_Figma.gif)
+
+<Caption>
+  The grid template is added to a frame via Design panel > Layout grid.
+</Caption>
+
+### Components
+
+Carbon for IBM.com components are available in the design kit.
+
+#### To bring components into your file:
+
+1. Have one or more of the Carbon for IBM.com libraries active.
+2. In the Layers panel, select `Layers → Assets`.
+3. Choose one of the Carbon for IBM.com themes.
+4. Select a component and drag it onto the canvas.
+
+![Using component variants](../../images/get-started/Designing_Components_Figma.gif)
+
+<Caption>
+  Components can be dragged from the Assets panel onto the canvas
+</Caption>
 
-We are currently working on creating separate kits for each theme so things may
-change but the alpha kit is great if you want to dive in and get started with
-Figma. Stay tuned for updates!
+#### Component variants
+
+Some components contain variants that allow you to change the properties of a component such as breakpoint, type, and interactive state. If a component contains variants, you will see them in the Design panel when you select a component. Before making updates directly to a component, check to see if the option is available as a variant.
```

Comment:
...you will see them in the `Design` panel when you select a component.

Is `Design` panel or Design panel? Either is fine, just need to be consistent. Referring to the code snippet styling.